Doc: Organized the Getting Started section (#39813)
grafana/grafana: docs/sources/introduction/oss-details.md

+++
title = "What is Grafana OSS"
weight = 5
aliases = ["/docs/grafana/latest/guides/what-is-grafana"]
+++

# What is Grafana OSS

> [Grafana open source software](https://grafana.com/oss/) allows you to query, visualize, alert on, and understand your data no matter where it’s stored. With Grafana you can create, explore and share all of your data through beautiful, flexible dashboards.

Reviewer: This is great. Can we use it in the _index file?

Author: I was trying to avoid duplicate content. Let's keep this as is, can always update the docs later.

> After you have [installed Grafana]({{< relref "../installation/_index.md" >}}) and set up your first dashboard using instructions in [Getting started with Grafana]({{< relref "../getting-started/getting-started.md" >}}), there are many things you can do next depending on your requirements. For example, if you want to view weather data and statistics about your smart home, then you might create a [playlist]({{< relref "../dashboards/playlist.md" >}}). If you are the administrator for a corporation and are managing Grafana for multiple teams, then you might need to set up [provisioning]({{< relref "../administration/provisioning.md" >}}) and [authentication]({{< relref "../auth/_index.md" >}}).

Reviewer: ```suggestion
After you have [installed Grafana]({{< relref "../installation/_index.md" >}}) and set up your first dashboard using our [Getting started with Grafana]({{< relref "../getting-started/getting-started.md" >}}), what you can do next depends on what you'd like to accomplish. For example, to view weather data and statistics about your smart home, you might create a [playlist]({{< relref "../dashboards/playlist.md" >}}). If you are the administrator for a corporation and managing Grafana for multiple teams, you might need to set up [provisioning]({{< relref "../administration/provisioning.md" >}}) and [authentication]({{< relref "../auth/_index.md" >}}).
```

Author: Incorporated most of the suggestions.

> The following sections provide an overview of things you might want to do with your Grafana instance and links to product documentation to help you learn more. For more guidance and ideas, check out the [Grafana Community forums](https://community.grafana.com/).

Reviewer: ```suggestion
In following sections, you can read an overview of what you can do with your Grafana instance. The overview provides links to product documentation to help you learn more. For more guidance and ideas, check out our [Grafana Community forums](https://community.grafana.com/).
```

Author: Updated content with your suggestion.
